### DISCUSSION TITLE
能否增加超星云盘？

### DISCUSSION THREAD

**Discussion by 1997buyinan** [AlistGo/alist#1364] [Ideas]
超星云盘是超星公司开发的云盘，有免费的100G空间，下载不限速，能到40M/s左右，可以直接预览
测试下载链接http://pan-yz.chaoxing.com/external/m/file/732193739423465472
在线登录https://passport2.chaoxing.com/login?fid=&newversion=true&refer=https://i.chaoxing.com云盘在左侧导航栏下方
客户端下载http://pan-yz.chaoxing.com/app/download

**Comment by kadong1** [comment]
超星的小组云盘是无限空间的。

**Comment by anwen-anyi** [comment]
V2有大佬pr来着，V3暂时没空就没pr

**Comment by youtehub** [comment]
来支持一波，希望能够集成这个网盘
